service/turn.rest/src/main/java/org/kaazing/gateway/service/turn/rest/TurnRestCredentials.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2007-2016, Kaazing Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kaazing.gateway.service.turn.rest;
+
+public interface TurnRestCredentials {
+
+    public String getUsername();
+    
+    public String getPassword();
+    
+    /**
+     * 
+     * @return long data type representing the duration for which the credentials are valid
+     * (time to live)
+     */
+    public long getTTL();
+    
+    public String getURIs();
+    
+    public String getResponseString();
```

Comment:
Have a utility method in a utility class that constructs that string.  This utility method/class will not go into the public API